WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -33,6 +33,9 @@
 
         // index of the last app-version
         LAST_APP_VERSION_INDEX,
+
+        // index of the last values stored for Stats Widget
+        STATS_WIDGET_ITEM_INDEX,
```

Comment:
This comment and the name of the variable are misleading, since it stores actual widget data rather than an index.